src/test/regress/output/multi_copy.source

```diff
@@ -1050,6 +1050,25 @@ SELECT create_distributed_table('tt1','id');
 \copy tt1 from STDIN;
 DROP TABLE tt1;
 END;
+-- Test dropping a column in front of the partition column
+CREATE TABLE newtable (col1 int, col2 int, col3 int, col4 int);
```

Comment:
Very minor, but `newtable` seems too generic, maybe `drop_copy_test_table` or so? (Yes, we have worse table names as well, so feel free to ignore)